core/assets/bundles/bundle.properties

```diff
@@ -1073,7 +1073,7 @@ tutorial.launch = Once you reach a specific wave, you are able to[accent] launch
 item.copper.description = The most basic structural material. Used extensively in all types of blocks.
 item.lead.description = A basic starter material. Used extensively in electronics and liquid transportation blocks.
 item.metaglass.description = A super-tough glass compound. Extensively used for liquid distribution and storage.
-item.graphite.description = Mineralized carbon, used for ammunition and highly conductive.
+item.graphite.description = Mineralized carbon, highly conductive and used as ammunition.
```

Comment:
item.graphite.description = Mineralized carbon is highly conductive and is often used as ammunition

Reply:
why does this need a review

Reply:
Because English-language is trash